src/fluree/db/dbfunctions/internal.cljc

```diff
@@ -58,8 +58,8 @@
                       (.getClass e')
                       msg)
             args' (if (coll? args) (str/join " " args) args)]
-        (format "Error in database function: %s: %s. Provided: %s"
-                function-name err-msg args'))
+        (clojure.core/str "Error in database function: " function-name ": "
```

Comment:
Any reason why full namespace is being used? If so, ignore.

Reply:
Yeah: https://github.com/fluree/db/blob/4a56d01f2b8a5a5e25b261bd4978c987ef8a7cf8/src/fluree/db/dbfunctions/internal.cljc#L2-L3